### ISSUE TITLE
Shutdown and Restart problem 

### ISSUE THREAD

**Issue by ps4rokx** [0x1iii1ii/PPPwn-Luckfox#63]
Don't know why every version I used to be but same restart and shutdown problem happened occasionally...

**Comment by PocketCoffee69** [comment]
Hey, same here, the chip (Luckfox pico mini B) is directly soldered onto the motherboard, every time i try to use luckfox the ps4 turns off after several attempts, i tried to reinitialize the PS4 but nothing.

Software version works just fine

**Comment by Nesquirt** [comment]
kernel panic.

**Comment by 0x1iii1ii** [comment]
did you all use the latest v1.2.2? try a clean install by re-burn my custom buildroot image in the release page and try again, let me know if the KP still happens

**Comment by PocketCoffee69** [comment]
I tried to do the procedure from scratch, without success, after several attempts the ps4 turns off.
software version still works fine

**Comment by 0x1iii1ii** [comment]
can you try to install v1.1.0 from the release page? after done report back and let me know if it work without KP.

**Comment by 0x1iii1ii** [comment]
use this cmd for install 
```
cd PPPwn-Luckfox
chmod +x install_NAND.sh
./install_NAND.sh
```

**Comment by PocketCoffee69** [comment]
tried version 1.1.0 without success

**Comment by alpay35** [comment]
I have the same problem cuh1004

**Comment by 0x1iii1ii** [comment]
> I have the same problem cuh1004

does the previous version work fine and no KP?

**Comment by PocketCoffee69** [comment]
out of nowhere, version 1.2.1 works flawlessly, slightly slow 5 minutes on average over 10 reboots, but no problems after that.
no web server feature, no big deal, but at the moment jailbreak works without problems.

**Comment by 0x1iii1ii** [comment]
if you don't want a web server try 1.1.0, it's more stable than the latest release 1.2.X since I did some optimization to minimize the package from buildroot. I just updated the source code you can try it and let me know its performance or keep using 1.2.1.

**Comment by PocketCoffee69** [comment]
Update:
With version 1.1.0 "new" it seems to be going better, always 10 restarts 10 successes, average time 2/3 minutes max, the only note is that during the jailbreak the PS4 controller disconnects, nothing serious, by pressing the PS button the controller reconnects without problems.

with this version it seems that the Luckfox shuts down correctly after completing the jailbreak

**Comment by 0x1iii1ii** [comment]
Did you burn the buildroot image I provided, or are you using stock image from Luckfox official? Also, does the PS4 controller disconnect every time or sometimes? I don't think it's because of the jailbreak process. I tested mine, and I never had such a weird problem.

**Comment by PocketCoffee69** [comment]
I used the buildroot you updated, the controller always disconnects, until it completes the jailbreak
with version 1.2.1 the controller does not disconnect, otherwise everything works correctly

**Comment by EVOXGSR** [comment]
Same.
Converting from Pi-Pwn to an internal Luckfox Pico Mini B.  
PS4 fw v9.0 has not once successfully been jailbroken.  
Just crashes after a couple of attempts with v1.2.1

**Comment by dave22z** [comment]
With version 1.1.0 "new" whtout web server i noticed that after success jailbreak the luckfox don't auto shutdown (i config it to auto shut down before)

**Comment by 0x1iii1ii** [comment]
> With version 1.1.0 "new" whtout web server i noticed that after success jailbreak the luckfox don't auto shutdown (i config it to auto shut down before)

did you try a fresh install? tested mine it works fine.

**Comment by 0x1iii1ii** [comment]
> Same. Converting from Pi-Pwn to an internal Luckfox Pico Mini B. PS4 fw v9.0 has not once successfully been jailbroken. Just crashes after a couple of attempts with v1.2.1

try the updated v1.1.0

**Comment by LouisGarry** [comment]
> > With version 1.1.0 "new" whtout web server i noticed that after success jailbreak the luckfox don't auto shutdown (i config it to auto shut down before)
> 
> did you try a fresh install? tested mine it works fine.

1.2.2 works fine here

**Comment by dave22z** [comment]
> > With version 1.1.0 "new" whtout web server i noticed that after success jailbreak the luckfox don't auto shutdown (i config it to auto shut down before)
> 
> did you try a fresh install? tested mine it works fine.

yes a fresh installation

**Comment by dave22z** [comment]
> > Same. Converting from Pi-Pwn to an internal Luckfox Pico Mini B. PS4 fw v9.0 has not once successfully been jailbroken. Just crashes after a couple of attempts with v1.2.1
> 
> try the updated v1.1.0

yes i'm with the 1.1.0 version because i don't need the web server just auto launch jailbreak then shutdown the luckfox

**Comment by dave22z** [comment]
> > > With version 1.1.0 "new" whtout web server i noticed that after success jailbreak the luckfox don't auto shutdown (i config it to auto shut down before)
> > 
> > 
> > did you try a fresh install? tested mine it works fine.
> 
> 1.2.2 works fine here

yes i noticed that the 1.2.2 version work good but i d'ont need the web server i think i will use it and disable web server option thank you all

**Comment by join1985** [comment]
Hi all, thanks to 0x1iii1ii for repository. After many tests I've solved the shutdown,rest mode and reboot problem.
I've ps4 slim with 11.00 firmware and luckfox pico plus.
I used this [https://github.com/DjPopol/EzPPPwn/releases/tag/EzPPPwn%23v1.21](url) stage1.bin (thanks to DjPopol) and this [https://github.com/EchoStretch/PPPwn/tree/8164f26167a3411810ae3b07e510e1f77da2b2b8/stage2](url) stage2 (thanks to EchoStretch)  I modified the stage2.c file to work with goldhen (tested with GoldHEN v2.4b17.2). (stage2.bin compiled and modified stage.c file attached).
Luckfox 1.1.0 (old) with install_NAND.sh modded by me (attached) I use pppwn file.
[stage1.bin.zip](https://github.com/user-attachments/files/16815073/stage1.bin.zip)
[stage2.bin.zip](https://github.com/user-attachments/files/16815075/stage2.bin.zip)
[install_NAND.sh.zip](https://github.com/user-attachments/files/16815079/install_NAND.sh.zip)
[stage2.c.zip](https://github.com/user-attachments/files/16815081/stage2.c.zip)

**Comment by joe97tab** [comment]
What is php version? 8.1 may miss some function. I think 8.2 ok.

**Comment by LouisGarry** [comment]
> Hi all, remove:
> 
> 111 // Enable UART 112 *(uint8_t *)kdlsym(uart_patch) = 0;
> 
> from stage1.c solve all problem with rest, shutdown and reboot problem!
> 
> source: https://github.com/TheOfficialFloW/PPPwn/blob/master/stage1/stage1.c
> 
> I'm on 11.00 luckfox mini b

Do you have the binary file?

**Comment by join1985** [comment]
> 
> Do you have the binary file?

[stage1.zip](https://github.com/user-attachments/files/17150001/stage1.zip)

Yes

**Comment by 0x1iii1ii** [comment]
> Hi all, remove:
> 
> 111 // Enable UART 112 *(uint8_t *)kdlsym(uart_patch) = 0;
> 
> from stage1.c solve all problem with rest, shutdown and reboot problem!
> 
> source: https://github.com/TheOfficialFloW/PPPwn/blob/master/stage1/stage1.c
> 
> I'm on 11.00 luckfox mini b

thank you for your finding, I'll test it to see if it actually fixes the problem, also @LouisGarry, please also report your result as well!

**Comment by join1985** [comment]
> > Hi all, remove:
> > 111 // Enable UART 112 *(uint8_t *)kdlsym(uart_patch) = 0;
> > from stage1.c solve all problem with rest, shutdown and reboot problem!
> > source: https://github.com/TheOfficialFloW/PPPwn/blob/master/stage1/stage1.c
> > I'm on 11.00 luckfox mini b
> 
> thank you for your finding, I'll test it to see if it actually fixes the problem, also @LouisGarry, please also report your result as well!

After 20 times work first shutdown problem 😞

**Comment by LouisGarry** [comment]
> > Hi all, remove:
> > 111 // Enable UART 112 *(uint8_t *)kdlsym(uart_patch) = 0;
> > from stage1.c solve all problem with rest, shutdown and reboot problem!
> > source: https://github.com/TheOfficialFloW/PPPwn/blob/master/stage1/stage1.c
> > I'm on 11.00 luckfox mini b
> 
> thank you for your finding, I'll test it to see if it actually fixes the problem, also @LouisGarry, please also report your result as well!

After trying 10 times, it looks the same as the original file.

**Comment by join1985** [comment]
> thank you for your finding, I'll test it to see if it actually fixes the problem, also @LouisGarry, please also report your result as well!

Problem partially solved!
before turning off the console, open the browser, with browser open hold down the ps button and rest/shutdown/reboot the console.
it works 100% of the time!
try it
